src/vanilla/Templates/ReadmeTemplate.cshtml

```diff
@@ -18,15 +18,15 @@ npm install @(Model.PackageName)
 ### Authentication, client creation and @(Model.GetSampleMethod()?.Name) @(Model.GetSampleMethodGroupName()) as an example.
 @EmptyLine
 ```javascript
-import * as msRest from "ms-rest";
-import { @(Model.Name), @(Model.ServiceModelsName) } from "@(Model.PackageName)";
-const subscriptionId = "<Subscription_Id>";
-const token = "<access_token>";
-const creds = new msRest.TokenCredentials(token);
-const client = new @(Model.Name)(creds, subscriptionId);
-@(Model.GenerateSampleMethod(true)).catch((err) => {
-  console.log('An error ocurred:');
-  console.dir(err, {depth: null, colors: true});
+const msRestAzure = require("ms-rest-azure");
+const @(Model.Name) = require("@(Model.PackageName)");
+msRestAzure.interactiveLogin().then((creds) => {
+    const subscriptionId = "<Subscription_Id>";
+    const client = new @(Model.Name)(creds, subscriptionId);
```

Comment:
There are some clients in Azure that are tenant based, for example the graph client (azure-graph) package. It would be nice to reflect on the constructor signature and instantiate the client accordingly in the example. This would save us from verifying it later when we review any automatic PR by the bot.